cmd/snap-confine/mount-support.c

```diff
@@ -321,9 +321,9 @@ static void sc_bootstrap_mount_namespace(const struct sc_mount_config *config)
 		// Fixes the following bugs:
 		//  - https://bugs.launchpad.net/snap-confine/+bug/1580018
 		//  - https://bugzilla.opensuse.org/show_bug.cgi?id=1028568
-		const char *dirs_from_core[] =
-		    { "/etc/alternatives", "/etc/ssl", "/etc/nsswitch.conf",
-			NULL
+		const char *dirs_from_core[] = {
+			"/etc/alternatives", "/etc/ssl", "/etc/nsswitch.conf",
+			"/etc/apparmor", "/etc/apparmor.d", NULL
```

Comment:
this probably merits a comment about:

some specifc and privileged interfaces (e.g docker-support) give access to apparmor_parser from the base snap which at a mimimum needs to use matching configuration 

and a pointer to the bug?